README.txt

```diff
@@ -24,17 +27,15 @@ Benefits
 Users browse, search, and submit entries into the knowledge base. Entries can disprove, support, replicate, or peer-review other entries. The website calculates and informs users the status of an entry's validity using this system. An entry is most meritable if it follows the scientific method correctly. An entry is least meritable if it is just an opinion, or comment, and is less visible when sorting. Lower merit entries should not be dismissed, however, because they often inspire more meritable entries to be submitted.
 
 
-Mission Statement
+Mission Statement:
+To teach and engage 7+ billion people in investigative science and critical thinking. To maximize collaboration and assist humanity in reaching its collective intellectual potential.
 
-To engage everyone in problem-solving, using the scientific method as the most meritable method. To ensure that, through Methodocracy, humanity always has the ability to problem solve as much and as fast as possible.
```

Comment:
I advise to avoid using generic language in a pitch. It does not make your pitch stand out. For instance, "everyone" will just shoot past people's ears. We can include the phrase "scientific method", but I highly recommend rewording everything else.